system-addon/lib/Screenshots.jsm

```diff
@@ -12,6 +12,8 @@ XPCOMUtils.defineLazyModuleGetter(this, "BackgroundPageThumbs",
   "resource://gre/modules/BackgroundPageThumbs.jsm");
 XPCOMUtils.defineLazyModuleGetter(this, "PageThumbs",
   "resource://gre/modules/PageThumbs.jsm");
+XPCOMUtils.defineLazyModuleGetter(this, "PageThumbsStorage",
+  "resource://gre/modules/PageThumbs.jsm");
```

Comment:
where are you using PageThumbsStorage?

Reply:
@sarracini the implementation changed a few times around and I forgot to remove it. Done. Thanks.